src/main/resources/templates/c_plus_plus/test/Tests.py

```
@@ -4,23 +4,67 @@
     testConfigure: TestConfigure = TestConfigure(".", buildDir,
-        ["-DCMAKE_BUILD_TYPE=Debug",
-         "-DCMAKE_CXX_FLAGS=-fsanitize=address",
-         "-DCMAKE_EXE_LINKER_FLAGS=-fsanitize=address"])
+                                                 ["-DCMAKE_BUILD_TYPE=Debug",
+                                                  "-DCMAKE_CXX_FLAGS=-fsanitize=address",
+                                                  "-DCMAKE_EXE_LINKER_FLAGS=-fsanitize=address"])
     tester.addTest(testConfigure)
-    tester.addTest(TestCompile(buildDir, "sort-test", requirements=[testConfigure.name], name="CompileSort"))
-    tester.addTest(TestCatch2(buildDir, "sort-test", ["CompileSort"]))
+
+    for test_name, test_path_name in test_filenames.items():
```

Ruff (0.15.5)

[warning] Loop control variable `test_path_name` not used within loop body

(B007)